src/ui/js/Overview.js

```diff
@@ -94,6 +94,14 @@ Overview.getOverview = function(callback) {
 Overview.showPage = function() {
   Overview.page = $('<div>');
 
+  var gameCountText;
+  var numGames=Api.new_games.nGames;
+  numGames+=Api.active_games.games.awaitingPlayer.length;
```

Comment:
Oh, i see the problem here - `Api.new_games` contains both new games which are awaiting your action *and* those which are awaiting the other player's action.  It's just like `Api.active_games` that way, and, as in that case, `Api.new_games.games.awaitingPlayer.length` is defined.  (However, for either array, i think you get undefined rather than 0 when there are no games, so you should probably not count on lazy casting to do the right thing, but instead define numGames as 0 and check whether each array is defined before adding its length, or something along those lines.)